src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -265,6 +265,9 @@ enum AppStrings {
 		static let resetDescription = NSLocalizedString("Settings_Reset_Description", comment: "")
 
 		static let navigationBarTitle = NSLocalizedString("Settings_NavTitle", comment: "")
+
+		static let onImageDescription = NSLocalizedString("Settings_onImageDescription", comment: "")
+		static let offImageDescription = NSLocalizedString("Settings_offImageDescription", comment: "")
```

Comment:
These two are pointing to non-existent strings. Settings -> NotificationSettings.

Reply:
Renamed "Settings..." to "NotificationSettings..." to align with the Localizable.strings files